Source/WebCore/rendering/RenderMenuList.cpp

```diff
@@ -322,12 +337,15 @@ LayoutRect RenderMenuList::controlClipRect(const LayoutPoint& additionalOffset)
 void RenderMenuList::computeIntrinsicLogicalWidths(LayoutUnit& minLogicalWidth, LayoutUnit& maxLogicalWidth) const
 {
     maxLogicalWidth = shouldApplySizeContainment() ? theme().minimumMenuListSize(style()) : std::max(m_optionsWidth, theme().minimumMenuListSize(style()));
-    maxLogicalWidth += m_innerBlock->paddingLeft() + m_innerBlock->paddingRight();
+    if (style().isHorizontalWritingMode())
+        maxLogicalWidth += m_innerBlock->paddingLeft() + m_innerBlock->paddingRight();
```

Comment:
paddingStart() + paddingEnd() does not work?

Reply:
Adopted.